ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -258,6 +258,8 @@ def install_lso_cluster(self):
 
         self.configure_encryption()
 
+        self.configure_data_protection()
```

Comment:
What about 4.13 case? I mean in the case we install 4.13 cluster out of master branch because of upgrade scenario.  Do we need to add condition to make sure that OCS version is >= 4.14 ?

Reply:
+1

Reply:
We don't run UI deployment on upgrade cases because of this condition... https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/ui/helpers_ui.py#L50

Reply:
I see this condition:
`ocs_version != ocp_version `

But our upgrade jobs do upgrade of both if it's Y-stream upgrade, we upgrade OCP,ODF 4.13 to OCP,ODF 4.14.

So in this case ODF and OCP is deployed from master - 4.14 branch the 4.13 version and then will do upgrade .

So the reply here is not considering this scenario and it's just one check to add here to avoid this code to run in 4.13 and below.